stf_functional_tests.yml

```diff
@@ -21,15 +21,20 @@
     - name: "Run collectd tests"
       import_tasks: tasks/test_collectd.yml
       tags:
-        - collectd-write-qdr-edge-only
-        - collectd-write-qdr-mesh
         - enable-stf
+        - stf-connectors-osp13
 
     - name: "Run metrics_qdr tests"
       import_tasks: tasks/test_qdr.yml
       tags:
-        - collectd-write-qdr-edge-only
-        - collectd-write-qdr-mesh
-        - ceilometer-write-qdr-edge-only
-        - ceilometer-write-qdr-mesh
         - enable-stf
+        - stf-connectors-osp13
+
+- name: "Run e2e tests"
+  hosts: undercloud-0
```

Comment:
I think you should add `ignore_errors: true` here, otherwise the tests could end prematurely when one of the e2e tests fails. I think this is what Emma describes in her top-level review comment.

Reply:
Thanks,Chris ! Will add it. Here is how output of ceph query looks like:
http://pastebin.test.redhat.com/960822
Basically we need to see if we get some metrics. In case when there is no metrics the query will return empty list and grep will fail.

Reply:
> I think you should add `ignore_errors: true` here, otherwise the tests could end prematurely when one of the e2e tests fails. I think this is what Emma describes in her top-level review comment.

Done.